pennylane/labs/vibrational/pes_generator.py

```diff
@@ -28,14 +29,16 @@
     _single_point,
 )
 
-# pylint: disable=too-many-arguments, too-many-function-args, c-extension-no-member, too-many-branches
-# pylint: disable= import-outside-toplevel, too-many-positional-arguments, dangerous-default-value
+# pylint: disable=too-many-arguments, too-many-function-args, too-many-branches
+# pylint: disable= import-outside-toplevel, too-many-positional-arguments
 
 # constants
-HBAR = 6.022 * 1.055e12  # (amu)*(angstrom^2/s)
-C_LIGHT = 3 * 10**8  # m/s
+# TODO: Make this code work in atomic units only.
+HBAR = (
+    sp.constants.hbar * (1000 * sp.constants.Avogadro) * (10**20)
+)  # kg*(m^2/s) to (amu)*(angstrom^2/s)
 BOHR_TO_ANG = 0.5291772106  # factor to convert bohr to angstrom
```

Comment:
Could this be `sp.constants.physical_constants["Bohr radius"][0] *1e10` ?

Reply:
Changed it.